packages/memo/src/memoize.ts

```diff
@@ -13,21 +17,46 @@ const isCachedValue = <A, R>(
 ): cachedValue is R => cache.has(arg) && cache.get(arg) === cachedValue;
 
 export const memoize = <T, A, R>(
-  fn: MemoizableFunction<T, A, R>
+  fn: MemoizableFunction<T, A, R>,
+  _options?: Options
 ): MemoizedFunction<T, A, R> => {
   const cache = new Map<A, R>();
+  const cacheTimers = new Map<A, ReturnType<typeof setTimeout>>();
 
   const memoized: MemoizedFunction<T, A, R> = function (this, arg) {
+    const cleanUp = (): void => {
+      cache.delete(arg);
+      cacheTimers.delete(arg);
+    };
+
     const cachedValue = cache.get(arg);
 
+    // return cachedValue if already cached
```

Comment:
I don't think we need these comments since it's well explained by the names of the functions

Reply:
Ok. Should I remove the comments in the `.spec.ts` file too?

Reply:
Commonly the comments related to the test file is the description itself, but I don't see any problem to have comments inside the test, if you want to!